packages/xo-web/src/xo-app/settings/acls/index.js

```diff
@@ -108,6 +108,9 @@ class AclTable extends Component {
       const { acls, roles } = this.state
       const resolvedAcls = filter(
         map(acls, ({ subject, object, action }) => ({
+          id: Math.random()
+            .toString(36)
+            .slice(2),
```

Comment:
The problem is that everytime `refresh` is called, each ACL will have a different ID which is bad. Also, it seems that ACLs already have an ID in xo-server so use that instead.